src/main/arm/nestedtemplates/adminTemplate.json

```diff
@@ -356,7 +363,7 @@
                "fileUris": [
                   "[uri(parameters('_artifactsLocationAdminTemplate'), concat('../scripts/', variables('name_scriptFile'), parameters('_artifactsLocationSasToken')))]"
                ],
-               "commandToExecute": "[concat('sh',' ',variables('name_scriptFile'),' ',parameters('wlsDomainName'),' ',parameters('wlsUserName'),' ',parameters('wlsPassword'),' ',parameters('adminVMName'),' ', variables('const_wlsHome'),' ')]"
+               "commandToExecute": "[concat('sh',' ',variables('name_scriptFile'),' ',parameters('wlsDomainName'),' ',parameters('wlsUserName'),' ',parameters('wlsPassword'),' ',parameters('adminVMName'),' ', variables('const_wlsHome'),' ',string(parameters('enableHTTPAdminListenPort')),' ')]"
```

Comment:
See https://github.com/wls-eng/arm-oraclelinux-wls/issues/158 Move `commandToExecute` properties that include passwords to `protectedSettings`